src/net/sourceforge/kolmafia/MonsterData.java

```diff
@@ -1858,6 +1860,31 @@ private String makeItemCount(MonsterDrop drop) {
     return drop.itemCount().isEmpty() ? "" : drop.itemCount() + " ";
   }
 
+  public String getFact() {
+    if (!(KoLCharacter.hasSkill(SkillPool.JUST_THE_FACTS)
+        && StandardRequest.isAllowed(RestrictedItemType.SKILLS, "Just the Facts"))) {
+      return null;
```

Comment:
returning "" will save a null check and simplify the method down to `buffer.append(monster.getFact())`

Reply:
I added the null because Gausie specifically mentioned it [here](https://github.com/kolmafia/kolmafia/pull/2707#issuecomment-2631488596)

I'll leave it for now until hopefully you two agree?

Reply:
I don't agree but don't care enough to block on it.